src/m365/aad/commands/o365group/o365group-add.ts

```diff
@@ -91,85 +100,35 @@ class AadO365GroupAddCommand extends GraphCommand {
           this.setGroupLogo(requestOptions, AadO365GroupAddCommand.numRepeat, resolve, reject, logger);
         });
       })
-      .then((): Promise<{ value: { id: string; }[] }> => {
-        if (!args.options.owners) {
-          if (this.debug) {
-            logger.logToStderr('Owners not set. Skipping');
-          }
-
-          return Promise.resolve(undefined as any);
-        }
-
-        const owners: string[] = args.options.owners.split(',').map(o => o.trim());
-
-        if (this.verbose) {
-          logger.logToStderr('Retrieving user information to set group owners...');
-        }
-
-        const requestOptions: any = {
-          url: `${this.resource}/v1.0/users?$filter=${owners.map(o => `userPrincipalName eq '${o}'`).join(' or ')}&$select=id`,
-          headers: {
-            'content-type': 'application/json'
-          },
-          responseType: 'json'
-        };
-
-        return request.get(requestOptions);
-      })
-      .then((res?: { value: { id: string; }[] }): Promise<any> => {
-        if (!res) {
+      .then((): Promise<any> => {
+        if (!ownerIds || ownerIds.length === 0) {
```

Comment:
If we agree to return an empty array from the `getUserIds` method, then we can simplify this check to just `ownerIds.length === 0`